geometry_msgs/Vector3[] vector
Header header
sensor_msgs/CompressedImage RGBImage
float32[] LSD_map
